src/labkey/ActionURL.ts

```diff
@@ -273,6 +273,16 @@ export interface ActionPath {
     controller: string;
 }
 
+/**
+ * decodeURI chooses not to decode encoded commas (presumably because encodeURI also does nothing with commas).
```

Comment:
This is [why](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURI#description):

<img width="799" alt="image" src="https://user-images.githubusercontent.com/3926239/208152670-02ebd879-e361-4e31-9513-1bed028716b5.png">

So should we consider any of these other characters? Also, just curious, where did you see that it is recommended to encode them?

Reply:
Here, for example: https://developer.mozilla.org/en-US/docs/Glossary/percent-encoding

Reply:
Thanks for that pointer.  I think in this usage, since we've already identified the part that's being decoded as a container path, which shouldn't include parts where these characters have meaning in the URI syntax, it should be safe to decode all of these characters (so they don't get double encoded later).  Thoughts?

Reply:
Ah, but we prevent the use of many of these characters in folder names. We allow ;, @, &, =, $, ,, and #. Perhaps we should expand the set of not allowed characters, but there may be some weird ones in the wild already.

Reply:
@labkey-nicka I've update the method to decode the other characters that are valid in project names but encoded by URI encoding. I believe returning the fully decoded path is what we want here since previously that is what was expected, just not being done.  Happy to discuss, though.

Reply:
I think what you've done works.